rootfs/var/lib/dietpi/services/dietpi-wifi-monitor.sh

```diff
@@ -12,12 +12,20 @@
 	[[ $TICKRATE =~ ^[1-9][0-9]*$ ]] || readonly TICKRATE=10
 	GATEWAY=
 
+	# Sleep without subprocess
+	# https://blog.dhampir.no/content/sleeping-without-a-subprocess-in-bash-and-how-to-sleep-forever
+	snore() {
+		local IFS
+		[[ -n "${_snore_fd:-}" ]] || { exec {_snore_fd}<> <(:); } 2>/dev/null
+		read ${1:+-t "$1"} -u $_snore_fd || :
+	}
```

Comment:
Nice idea, we could even use this throughout all our scripts as a `dietpi-globals` function. However, there is a lot over strange overhead insight I'm not sure why it's done like that. Here a much simpler solution, functional according to my tests:
```suggestion
	snore(){ read -t "${1:-1}" < /dev/zero; }
```

Reply:
Just do a quick test, your version of snore make bash run full tilt (100% CPU). I guess because _read_ success reading NULs, discards them all and then do the whole thing again.

Reply:
Ah you're right, another approach is:
```suggestion
	snore(){ read -t "${1:-1}" <> <(:); }
```
This however is quite close to yours already. Not sure whether it's worth it to keep the file descriptor open and re-use it on subsequent function calls.

Reply:
I think if this function is to be reused everywhere, a few bytes to keep a file descriptor alive is worth it, compare to creating a new one.

Reply:
I think you're right. I read about the `{var}<>` syntax now, pretty nice, I have a few other possible use cases for this 🙂.